library/common-library/templates/_security-context.tpl

```diff
@@ -0,0 +1,33 @@
+{{- /* Defines the container securityContext context */ -}}
+{{- define "common.securityContext.container" -}}
+    {{- if .Values.global -}}
+        {{- if .Values.global.securityContext -}}
```

Comment:
logic is currently bit cumbersome due to the nested cases. I lend it of over to @kang-makes to be improved 😄

Reply:
Solved in [#f27f4e9](https://github.com/newrelic/helm-charts/pull/725/commits/f27f4e99b33395141d632585fcd4c869f41c9ae1)